src/app/components/progress-indicator/waterfall-progress-indicator/index.html

```diff
@@ -0,0 +1,325 @@
+          <sky-docs-anatomy-item anatomyNumber="5">Waterfall title</sky-docs-anatomy-item>
+          <sky-docs-anatomy-item isOptional="true" anatomyNumber="6">Step details</sky-docs-anatomy-item>
+          <sky-docs-anatomy-item isOptional="true" anatomyNumber="7">Start over action</sky-docs-anatomy-item>
+        </sky-docs-anatomy>
+      </sky-demo-page-content>
+    
+      <sky-demo-page-content sectionHeading="Options">
+        <h3>Waterfall title</h3>
+        <div class="sky-text-block">
+          <p>We recommend including a title to reflect the context of the overall task. The title does not change when users progress through the waterfall steps.</p>
+        </div>
+        
+        <h3>Step details</h3>
+        <div class="sky-text-block">
+          <p>You can provide additional details about completed steps, but limit details to single pieces of information. Don’t go overboard because users can reopen modals to review choices as necessary.</p>
+        </div>
+        
+        <h3>Start over action</h3>
+        <div class="sky-text-block">
+          <p>You can include a start over action to let users erase all saved data from the waterfall process. This returns user to the first step or to an earlier point in the workflow if decisions occured before the waterfall progress indicator.</p>
+        </div>
+      </sky-demo-page-content>
+    
+      <sky-demo-page-content sectionHeading="Behavior & states">
+        <div class="sky-text-block">
+          <p>Waterfall progress indicators guide users through one active step at a time. When users complete a step, the progress indicator updates its step indicator and step action to the completed state and updates the next step indicator and step action to the active state. Users can review and edit completed steps, but this does not undo the completed state.</p>
+          <p>The final step summarizes the changes and requests user confirmation. You do not need to include all data that users enter in the summary, but confirmation is required. After users complete the final step, replace the waterfall progress indicator with a page that displays the finished or active state of the system. Any page type is allowed, including record pages, work centers, and list pages.</p>
+        </div>
+      </sky-demo-page-content>
+    
+      <sky-demo-page-content sectionHeading="Content">
+        <div class="sky-text-block">
+          <p>Step labels in waterfall progress indicators start with a number and concisely describe the objective. Use the <b>&lt;verb&gt; &lt;direct object&gt;</b> format. In the label for the last step, describe the cumulative change that occurs at the end of the process.</p>
+          <p>Step action labels closely follow step labels but do not need to be identical. For the first step action label, be generic and conversational (“Get started”), and for completed step actions, use verbs appropriate to post hoc actions (“Change,” “Edit”).</p>
```

Comment:
What is the preferred format for action labels? Should they also follow the <verb> <direct object> format that we recommend for the step labels? Users could infer this pattern from the previous paragraph or assume that because we don’t state it explicitly for actions, it doesn’t apply here. I’d remove the ambiguity by being explicit. If this rule applies to both step labels and step actions, I’d make that clear in the first paragraph and then either update or remove the examples for the action labels since they don’t include direct objects. But if it doesn't apply here, I'd explicitly state the recommended format in this paragraph.

Reply:
Ported over to #625.